readme.md

```diff
@@ -34,6 +34,7 @@ Blockchains created with Ignite CLI use the [Cosmos SDK](https://github.com/cosm
 
 | Ignite CLI | Cosmos SDK | IBC                  | Notes                                            |
 | -------- | ---------- | -------------------- | ------------------------------------------------ |
+| v0.20.4  | v0.45.2    | v3.0.0               | |
```

Comment:
@ilgooz did we want to include newer versions here? what about IBC?

Reply:
No, not for this release.